src/main/java/apoc/export/graphml/XmlGraphMLWriter.java

```diff
@@ -133,7 +137,7 @@ private int writeProps(XMLStreamWriter writer, PropertyContainer node) throws IO
 
     private void writeData(XMLStreamWriter writer, String prop, Object value) throws IOException, XMLStreamException {
         writer.writeStartElement("data");
-        writer.writeAttribute("key", prop);
+        writer.writeAttribute("key", prop.equals("labels") ? "label" : prop);
```

Comment:
Non capisco perché facciamo questo: `prop.equals("labels")`
E' necessario?
Non possiamo passare direttamente "label" e rimetterlo come prima?
Soprattutto io, come fatto per i json, passerei `labels` in caso dei nodi e `label` in caso delle relazioni.

Reply:
E' un cambiamento grosso rispetto a com'è adesso?

Reply:
Facciamo questo controllo perchè altrimenti se mettiamo un nome nell'`attr.name` diverso da `label` il dato non sarà visualizzato nella colonna esistente `Label`. Eventualmente potremmo cambiare le key in questo modo: 
`<key id="labels" for="node" attr.name="label" attr.type="string"/>`
`<key id="label" for="edge" attr.name="label" attr.type="string"/>`